src/js/components/visualization/ChartComponent.jsx

```diff
@@ -114,7 +114,8 @@ export class ChartComponent extends Component {
 
           this._chart.drilldown({
             ...chartData,
-            data: (seriesItem || dataItem)[drillDownFieldName]
```

Comment:
can you refactor this so its a little more clear? i would probably just put it before the object definition and just figure it out in a slightly more verbose way.